client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -27,11 +27,86 @@
         style="max-height: 52vh;"
       />
     </b-form-group>
+
+    <div>
+      <b-form-group
+        :label="$t('sanitizers.label')"
+        label-size="lg"
+      >
+        <template #label>
+          <div
+            class="d-flex"
+          >
+            Condition Fields
+
+            <b-button
+              variant="link"
+              class="p-0 ml-1 mr-auto"
+              @click="add"
+            >
+              + Add
+            </b-button>
+
+            <b-button
+              variant="link"
+              :href="`${documentationURL}#value-sanitizers`"
+              target="_blank"
+              class="p-0 ml-1"
+            >
+              Examples
+            </b-button>
+          </div>
+        </template>
+      </b-form-group>
+
+      <b-table-simple
+        v-if="block.options.fieldConditions.length"
+        borderless
+      >
+        <b-tbody>
+          <b-tr
+            v-for="(condition, i) in block.options.fieldConditions"
+            :key="i"
+          >
+            <b-td>
+              <vue-select
+                v-model="condition.field.name"
+                :options="fieldOptions"
+                class="bg-white"
+                @close="(() => $root.$emit('checkState', { items: options.fieldConditions, indicator: 'field' }))"
+              />
+            </b-td>
+
+            <b-td>
+              <b-input-group>
+                <b-input-group-prepend>
+                  <b-button variant="dark">
+                    ƒ
+                  </b-button>
+                </b-input-group-prepend>
+                <b-form-input
+                  v-model="condition.condition"
+                />
+              </b-input-group>
+            </b-td>
+
+            <b-td>
+              <c-input-confirm
+                class="mt-2 ml-2"
```

Comment:
no margin here, you probably want to align it, use the proper td/tr styles for that. Look at the module edit